docs/index.md

```diff
@@ -35,3 +35,39 @@ Each abstraction is built over the lower-level one, and client code
 can mix and match abstractions very easily: you may be able to use
 `SimplePubsub` for most of your code, dropping down to
 `PublisherClient` and `SubscriberClient` occasionally, for example.
+
+## Google.Datastore.V1Beta3
+
+A library for working with [Cloud Datastore](https://cloud.google.com/datastore/).
+
+Sample code:
+
+```csharp
+var client = DatastoreClient.Create();
+
+var projectId = ...;
+var partitionId = new PartitionId { ProjectId = projectId };
+var entity = new Entity
+{
+    Key = new Key { PartitionId = partitionId, Path = { new PathElement { Kind = "message" } } },
+    ["created"] = DateTime.UtcNow,
+    ["text"] = "Text of the message"
+};
+var transaction = client.BeginTransaction(projectId).Transaction;
+var commitResponse = client.Commit(projectId, Mode.TRANSACTIONAL, transaction, new[] { entity.ToInsert() });
+var insertedKey = commitResponse.MutationResults[0].Key;
+```
+
+See [`DatastoreClient`](obj/api/Google.Datastore.V1Beta3.DatastoreClient.yml)
```

Comment:
All the .yml links on https://github.com/GoogleCloudPlatform/gcloud-dotnet/blob/master/docs/index.md fail.
E.g.
https://github.com/GoogleCloudPlatform/gcloud-dotnet/blob/master/docs/obj/api/Google.Pubsub.V1.PublisherClient.yml
https://github.com/GoogleCloudPlatform/gcloud-dotnet/blob/master/docs/obj/api/Google.Pubsub.V1.Subscriber.ISubscriberClient.yml